### PR TITLE
Afzaledx/sol 1653 course due date for self paced

### PR DESCRIPTION
**Pull request by ibrahimahmed443**

### CODE REVIEW

File: edx_proctoring/api.py [hunk 1]
```diff
@@ -1779,6 +1779,16 @@ def get_student_view(user_id, course_id, content_id,
...
 
+        # Just in case the due date has been changed because of the
+        # self-paced courses, use the due date from the context and
+        # update the local exam object if necessary.
+        if exam['due_date'] != context.get('due_date', None):
+            update_exam(
+                exam_id=exam['id'],
+                due_date=context.get('due_date', None)
+            )
+        exam['due_date'] = context.get('due_date', None)
```
Comment (douglashall) [inline edx_proctoring/api.py]: Move this inside the if block.

**Comment by schenedx** [comment]
Agree with @douglashall's comment. Rest LGTM

File: edx_proctoring/tests/test_api.py [hunk 2]
```diff
@@ -1385,7 +1385,8 @@ def test_get_studentview_submitted_timed_exam_with_past_due_date(self, due_date,
             context={
                 'is_proctored': False,
                 'display_name': self.exam_name,
-                'default_time_limit_mins': 10
+                'default_time_limit_mins': 10,
+                'due_date': due_date
```
Comment (bderusha) [inline edx_proctoring/tests/test_api.py]: NIT: please use trailing comma

**Comment by douglashall** [comment]
:+1:

**Pull request merged**